bundle-size/app.js

```diff
@@ -228,33 +228,41 @@ module.exports = app => {
   app.on('pull_request_review.submitted', async context => {
     const approver = context.payload.review.user.login;
     const owners = await getOwners(context.github);
+    const pullRequestId = context.payload.pull_request.number;
+    const headSha = context.payload.pull_request.head.sha;
 
     if (context.payload.review.state == 'approved' &&
         owners.includes(approver)) {
-      context.log(`Pull request ${context.payload.pull_request.number} ` +
-          'approved by a bundle-size keeper');
+      context.log(`Pull request ${pullRequestId} approved by a bundle-size ` +
+          'keeper');
```

Comment:
Right now, we have `bundle-size keeper` and `bundle-size squad`, correct?

Future task: Explain this terminology in the details page for the check.

Reply:
Approval here means approving the PR itself, not specifically the check

Reply:
Created an issue for the future task: #30